modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_1/UpgradeDDMFormInstance.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.internal.upgrade.v4_3_1;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONException;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * @author Rodrigo Paulino
+ */
+public class UpgradeDDMFormInstance extends UpgradeProcess {
```

Comment:
I don't mind it. Would Brian accept that?

Reply:
I suggested it because I saw that there is a class called `UpgradeDDMFormInstanceSettings`, which performs the same update (`update DDMFormInstance set settings_ ...`), so I believe it is not a problem. 
But we can keep it that way so we don't have to resend the PR again 😝.

Reply:
But don't I have to send this again to make your suggested changes? If you want, I can throw this also in.

Reply:
Sorry, I meant resend to Brian @rodrigopaulino .

Reply:
Ok. Just to be clear, now. 😅 Do I leave this as is, or should I change it to `UpgradeDDMFormInstanceSettings`?

Reply:
You can leave it like this 🤣